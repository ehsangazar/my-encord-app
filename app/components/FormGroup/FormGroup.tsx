interface FormGroupProps {
  children: React.ReactNode;
}

const FormGroup = ({ children }: FormGroupProps) => {
  return <div className="flex my-2">{children}</div>;
};

export default FormGroup;
