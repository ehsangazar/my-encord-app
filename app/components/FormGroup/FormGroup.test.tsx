import renderer from "react-test-renderer";

import FormGroup from "./FormGroup";

it("FormGroup component should render correctly", () => {
  const tree = renderer.create(<FormGroup>Click me</FormGroup>).toJSON();
  expect(tree).toMatchSnapshot();
});
