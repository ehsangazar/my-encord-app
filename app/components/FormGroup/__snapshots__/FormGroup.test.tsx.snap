// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormGroup component should render correctly 1`] = `
<div
  className="flex my-2"
>
  Click me
</div>
`;
